rules/id/id-jawi.js

```diff
@@ -0,0 +1,50 @@
+﻿( function ( $ ) {
+	'use strict';
+	var jawiKeyboard = {
+		id: 'id-jawi',
+		name: 'Jawi keyboard',
+		description: 'Jawi keyboard for Indonesian Wikisource',
+		date: '2023-09-21',
+		URL: 'https://www.mediawiki.org/wiki/Help:Extension:UniversalLanguageSelector/Input_methods/jawi-keyboard',
```

Comment:
Need to create this page. Not a blcoker for the PR though

Reply:
done
https://www.mediawiki.org/wiki/Help:Extension:UniversalLanguageSelector/Input_methods/jawi-keyboard

Reply:
Perhaps the on-wiki documentation need to be updated as well to reflect the changes in the PR.

Reply:
done